src/vdaf.rs

```diff
@@ -220,6 +220,23 @@ where
     Finish(V::OutputShare),
 }
 
+impl<V: Aggregator<L>, const L: usize> PrepareTransition<V, L>
+where
+    for<'a> &'a V::AggregateShare: Into<Vec<u8>>,
+{
+    /// Return the output share.
+    ///
+    /// # Panics
+    ///
+    /// This method panics if processing is not complete.
+    pub fn unwrap_finish(self) -> V::OutputShare {
```

Comment:
This appears to only be used in a doctest. Can you restrict it to `[cfg(test)]` or `[cfg(doc)]` or whichever is appropriate?

Reply:
It seems icky to not export something in the public API that's used in a doctest.

Reply:
I see your point, but exporting this from the public API implies that it's intended for usage by clients of libprio which isn't really true. Since you only use it one place, could you delete the method and "inline" the `match` block into the doctest on `struct Prio3`,

Reply:
Done!